Workbooks/PostgreSQL Flexible Server/Performance/High Temporary Files/High Temporary Files.workbook

```diff
@@ -2324,12 +2382,12 @@
       "type": 3,
       "content": {
         "version": "KqlItem/1.0",
-        "query": "let graphgrain=iff({TimeRange:grain}<15m,15m, {TimeRange:grain});\r\nlet TimeStart = todatetime('{GlobalStartTime}');\r\nlet TimeEnd = todatetime('{GlobalEndTime}');\r\n{QueryStoreTable}\r\n| where UserId != 10\r\n| where QueryId == '{QueryIdsByTemp}'\r\n| where EndTime >= TimeStart and EndTime <= TimeEnd\r\n| distinct MeanExecDurationMs,MaxExecDurationMs,MinExecDurationMs,Calls,QueryId,QueryType,TempBlksRead,TempBlksWritten\r\n| extend Mean_Temp_blks_written = TempBlksWritten* 8 / Calls\r\n| extend Mean_Temp_blks_read = TempBlksRead * 8 / Calls\r\n| summarize MeanTempBlksWritten = avg(Mean_Temp_blks_written), AvgExecutionTime=avg(MeanExecDurationMs), MaxTempBlksWritten = max(Mean_Temp_blks_written), MaxExecutionTime=max(MaxExecDurationMs), TotalCalls=sum(Calls), MinTempBlksWritten=min(Mean_Temp_blks_written), MinExecutionTime=min(MinExecDurationMs) by QueryId,QueryType",
+        "query": "let graphgrain=iff({TimeRange:grain}<15m,15m, {TimeRange:grain});\r\nlet TimeStart = todatetime('{GlobalStartTime}');\r\nlet TimeEnd = todatetime('{GlobalEndTime}');\r\n{QueryStoreTable}\r\n| where UserId != 10\r\n| where QueryId == '{QueryIdsByTemp}'\r\n| where EndTime >= TimeStart and EndTime <= TimeEnd\r\n| distinct MeanExecDurationMs,MaxExecDurationMs,MinExecDurationMs,Calls,QueryId,QueryType,TempBlksRead,TempBlksWritten,RuntimeStatsEntryId\r\n| extend Mean_Temp_blks_written = TempBlksWritten* 8 / Calls\r\n| extend kb_Temp_blks_written = TempBlksWritten * 8\r\n| summarize MeanTempBlksWritten = avg(Mean_Temp_blks_written), MeanExecutionTime=avg(MeanExecDurationMs), MaxTempBlksWritten = max(Mean_Temp_blks_written), MaxExecutionTime=max(MaxExecDurationMs), TotalCalls=sum(Calls), TotalTempBlksWritten=sum(kb_Temp_blks_written), MinTempBlksWritten=min(Mean_Temp_blks_written), MinExecutionTime=min(MinExecDurationMs) by QueryId,QueryType",
         "size": 4,
-        "title": "QueryId {QueryIdsByTemp} summary in selected TimeRange",
+        "title": "QueryId {QueryIdsByTemp} Summary in Selected TimeRange",
         "showExportToExcel": true,
         "queryType": 0,
-        "resourceType": "microsoft.dbforpostgresql/flexibleservers",
+        "resourceType": "microsoft.resources/resourcegroups",
```

Comment:
why this change? Wount be this a regression?

Reply:
![image](https://user-images.githubusercontent.com/127991103/236349391-34fb898c-1f3f-40b0-8346-f8780dfdcf2e.png)



Corrected